kura/distrib/pom.xml

```
@@ -1295,10 +1300,10 @@
                                         <property name="project.version" value="${project.version}" />
                                         <property name="project.build.profile" value="${project.build.profile}" />
                                         <property name="project.build.directory" value="${project.build.directory}" />
-                                        <property name="build.name" value="generic-arm32-nn" />
-                                        <property name="target.device" value="generic-arm32-nn" />
+                                        <property name="build.name" value="armhf-nn" />
+                                        <property name="target.device" value="armhf-nn" />
                                         <property name="native.tag" value="armv6hf" />
-                                        <property name="kura.os.version" value="raspbian" />
+                                        <property name="kura.os.version" value="linux" />
```

Nitpicking here but, we could/should probably put `debian` in here since that is our main target.

```suggestion
                                        <property name="kura.os.version" value="debian" />
```

---

Agree. In the future we should consider removing this property since it is not used anywhere...

---

Addressed in [f825eb8](https://github.com/eclipse-kura/kura/pull/5609/commits/f825eb8b27589f963cd5229be16bd30c1b5ed0d9)